conf/messages.en

```diff
@@ -48,12 +48,33 @@ signedOut.guidance = We did not save your answers.
 unauthorised.title = You can’t access this service with this account
 unauthorised.heading = You can’t access this service with this account
 
+monthlyreturns.dateConfirmNilPayments.title = Confirm nil payments to subcontractors this month
+monthlyreturns.dateConfirmNilPayments.heading.h1 = Confirm nil payments to subcontractors this month
+monthlyreturns.dateConfirmNilPayments.heading.h2 = Enter the tax month and year you are reporting nil payments for
+monthlyreturns.dateConfirmNilPayments.paragraph = A nil return is to record that you have not made any payments to subcontractors in the Construction Industry for the month shown below.
```

Comment:
monthlyreturns.dateConfirmNilPayments.paragraph = A nil return is to record that you have not made any payments to subcontractors in the Construction Industry Scheme for the tax month shown below.

Reply:
Changed.